oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/util.js

```diff
@@ -45,12 +60,90 @@ export function createFormAppInit({
   autoInit = true,
   ContainerComponent = React.Fragment,
   componentOverrides = {},
+  recordSerializerProp,
```

Comment:
replace with `...props` & keep the later initialization code as similar to DepositFormApp init

Reply:
I'd avoid naming props ***prop :)